libs/blocks/table-of-contents/table-of-contents.js

```diff
@@ -16,7 +16,7 @@
 
 import { createTag, getConfig } from '../../utils/utils.js';
 
-const DOWN_ARROW_ICON = '<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="10.875" height="13.323" viewBox="0 0 10.875 13.323"><title>down arrow</title><g fill="none" stroke="#1473e6" stroke-linecap="round" stroke-width="2"><path d="M5.478 1v10.909"/><g><path d="m9.461 7.885-4.023 4.023m0 0L1.414 7.885"/></g></g></svg>';
+const DOWN_ARROW_ICON = '<svg aria-hidden="true" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="10.875" height="13.323" viewBox="0 0 10.875 13.323"><title>down arrow</title><g fill="none" stroke="#1473e6" stroke-linecap="round" stroke-width="2"><path d="M5.478 1v10.909"/><g><path d="m9.461 7.885-4.023 4.023m0 0L1.414 7.885"/></g></g></svg>';
```

Comment:
Twice the hidden, twice the accessibility! 😀 Let's remove the duplicate.

Reply:
Fixed 👍